### PR TITLE
Update kubevirt.github.io container Dockerfile

### PR DESCRIPTION
**Pull request by cwilkers**
- Bump Fedora from 34 to 36
- Fix packages diff between versions
- Go with system python 3.10 from Fed 36
- Fix casperjs dependency issue

Signed-off-by: cwilkers <cwilkers@redhat.com>

### CODE REVIEW

**Comment by cwilkers** [comment]
@dhiller If you could, I'd appreciate your review here, as I'd like to make sure I'm not introducing some unexpected side effects into the CI/CD flow.

**Review by brianmcarey — COMMENTED** [review]
/lgtm

**Comment by dhiller** [comment]
I think we are currently missing automatic presubmit check ([example for userguide](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml#L143)) and image push ([userguide example](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml#L148)) as we have on the other images. This way we would not need to push manually, while also having a check on any change of the image.

@cwilkers are you up to doing this? @brianmcarey maybe you could support him?

**Comment by cwilkers** [comment]
> @cwilkers now that I took a look at the jobs, I don't see any reference to this image?

@dhiller That is surprising, but a bit of a relief. As far as I understand, the image is there to provide a local platform for rendering and testing the website. If it is completely detached from the CI/CD process, perhaps it makes more sense to move the Dockerfile out of project-infra and into kubevirt.github.io.

**Comment by dhiller** [comment]
@cwilkers I think your idea to use a common image for both user-guide and kubevirt.github.io is great! Also I think that this image can stay very well in project-infra below images, as it then will actually be used by our jobs, and then deserves a build-image and publish-image job, so that it gets tested, updated and published whenever it changes. (Also it spares you the hassle of manually publishing it, and it will be part of the automated image updates for the prow jobs)

File: github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml [hunk 1]
```diff
@@ -179,6 +179,37 @@ postsubmits:
                 memory: "1Gi"
               limits:
                 memory: "1Gi"
+    - name: publish-kubevirt-kubevirt.github.io-image
```
Comment (dhiller) [inline github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml:182]: Would you mind creating a build presubmit similar to [this one](https://github.com/kubevirt/project-infra/blob/e66f3b49d7c712a5aa1ff35f72c1142dce1690cc/github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-presubmits.yaml#L63) for your image? This way we would catch failures in the image build before we merge.

**Review by dhiller — COMMENTED** [review]
Looks good already, one comment inline.

File: github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml (hunk 1 above)
Comment (cwilkers) [reply github/ci/prow-deploy/files/jobs/kubevirt/project-infra/project-infra-postsubmits.yaml:182]: No problem, done.

**Review by dhiller — APPROVED** [review]
/approve

**Comment by dhiller** [comment]
@cwilkers I have started the publish job manually: https://prow.ci.kubevirt.io/view/gs/kubevirt-prow/logs/publish-kubevirt-kubevirt.github.io-image/1560261254461788160#1:build-log.txt%3A63

There seems to be a problem in the Dockerfile. Would you take a look?

**Pull request merged**